cmake/catkin_libraries.cmake

```diff
@@ -144,16 +147,17 @@ macro(catkin_replace_imported_library_targets VAR)
           endforeach()
         endif()
         foreach(imp_lib ${imported_libraries})
-          list(APPEND ${VAR} "${imp_lib}")
+          list(APPEND result "${imp_lib}")
         endforeach()
       else()
         # not an imported library target
-        list(APPEND ${VAR} "${lib}")
+        list(APPEND result "${lib}")
       endif()
     else()
-      list(APPEND ${VAR} "${lib}")
+      list(APPEND result "${lib}")
     endif()
   endforeach()
+  set(${VAR} "${result}" PARENT_SCOPE)
   #set(_argn ${ARGN})
   #debug_message(10 "catkin_replace_imported_library_targets(${VAR} ${_argn}) ${${VAR}}")
```

Comment:
Just to be safe if someone enables this line in the future can you also update it to use `${result}` instead of `${${VAR}}`.